pkg/resources/secondary_connection.go

```diff
@@ -0,0 +1,190 @@
+package resources
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/helpers"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/internal/provider"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/schemas"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/customdiff"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var secondaryConnectionSchema = map[string]*schema.Schema{
+	"name": {
+		Type:             schema.TypeString,
+		Required:         true,
+		ForceNew:         true,
+		Description:      blocklistedCharactersFieldDescription("String that specifies the identifier (i.e. name) for the connection. Must start with an alphabetic character and may only contain letters, decimal digits (0-9), and underscores (_). For a secondary connection, the name must match the name of its primary connection."),
+		DiffSuppressFunc: suppressIdentifierQuoting,
+	},
+	"is_primary": {
+		Type:        schema.TypeBool,
+		Computed:    true,
+		Description: "Indicates if the connection has been changed to primary. If change is detected, the secondary connection will be recreated.",
```

Comment:
Why is there no `enable_failover_to_accounts`?

Reply:
im not sure if there should be, secondary connection does not serve as primary one, to me it shouldn't have the list of accounts that could replicate it, since it cannot be replicated (unless promoted to primary)

Reply:
Okay - I was just asking.